draw/viewport.py

```diff
@@ -73,9 +74,10 @@ def __init__(self, engine, context, scene):
         self._normal_file_path = self._make_denoiser_filepath("normal")
         self._denoised_file_path = self._make_denoiser_filepath("denoised")
         current_dir = os.path.dirname(os.path.realpath(__file__))
-        self._denoiser_path = os.path.join(os.path.dirname(current_dir), "bin", "denoise")
-        if platform.system() == "Windows":
-            self._denoiser_path += ".exe"
+        self._denoiser_path = which(
+            "denoise",
```

Comment:
Shouldn't this be `oidnDenoise` as well?

Reply:
Fixed.